dialog-db/starting-dialog.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dialog - Personal Substrate for Cooperative Computing
+date: 2025-08-12
+---
+
+# Dialog - Personal Substrate for Cooperative Computing
+
+You're planning dinner for friends this weekend. Your calendar knows who's coming. Your contacts know Sarah's gluten-free, Tom's vegetarian. As you open a meal planner, it suggests recipes everyone can enjoy, portions adjusted for your group. Your shopping list populates with needed ingredients.
+
+No configurations. No authorizations. No information retrieval from corporate silos. Your tools cooperate through a shared substrate—each deriving insights and contributing new understanding. They complement each other, becoming true [bicycles for your mind][].
+
+## Means of Cooperation
+
+Today's tools cooperate through following coordination models:
+
+**Corporate suites** (Google Workspace, Microsoft Office) work together because one company controls everything.
+
+**Limited integrations** where companies choose which APIs to expose. Slack decides who to integrate with, not you. You can't message Signal friends from WeChat.
+
+**Manual coordination** for everything else—you copy data between tools, maintain consistency, context-switch endlessly.
+
+Your contacts live in one app, dietary preferences in another, calendar events in a third. You are the integration layer.
+
+## The Paradigm Shift
+
+Large Language Models enable anyone to generate specialized tools. A food blogger creates a meal planner. A fitness coach builds a nutrition tracker. Each speaks its own language.
+
+This renders existing cooperation models completely inadequate:
+- No corporation controls thousands of indie tools (hopefully)
+- Too many tools for each one to integrate with the rest
+- Manual coordination becomes impossible
+
+When data fragments across origins, cooperation requires concerted effort from all parties. Will these tools trap your data in new silos? Or enable emergent cooperation on a substrate you control?
+
+## Dialog: The Solution
+
+Dialog is a knowledge substrate—your personal context that tools derive insights from and contribute back to. Built on semantic facts, everything becomes queryable:
```

Comment:
Imagine a database that changes shape to fit your needs. Go beyond rigid schemas to select just the data you need, when you need it, in the form that suits you. 

Something something go beyond graphQL.